docs/gallery_conf.py

```diff
@@ -0,0 +1,20 @@
+import os
+import sys
+
+import plotly.io as pio
+pio.renderers.default = 'sphinx_gallery'
```

Comment:
what does this do?

Reply:
It seems like it sets the default renderer for Plotly... since i am not using it, I removed it. It is included because I used the config script that is in the mkdocs gallery as starting point for mine. this is actually useless for us